### PR TITLE
Update ec_si.py

### PR DESCRIPTION
**Pull request by DanielNwaeze**
Modified based on latest cli changes for commit

### CODE REVIEW

File: fiboa_cli/datasets/ec_si.py [hunk 1]
```diff
@@ -40,15 +45,14 @@
     'COLOR': 'color', #fiboa custom field
     'EC_trans_n': 'EC_trans_n', #fiboa custom field
     'EC_hcat_n': 'EC_hcat_n', #fiboa custom field
-    'EC_hcat_c': 'EC_hcat_c' #fiboa custom field
+    'EC_hcat_c': 'EC_hcat_c', #fiboa custom field
+    'EC_NUTS3': 'EC_nuts3', #fiboa custom field
```
Comment (m-mohr) [inline fiboa_cli/datasets/ec_si.py:49]: If other EC datasets also use this field, maybe keep it uppercase?

**Review by DanielNwaeze — COMMENTED** [review]
Modified the change @m-mohr

**Comment by m-mohr** [comment]
While you are at it, would you mind adding a test similar to the ones we have?

Will merge nevertheless now, changes look good.

**Comment by DanielNwaeze** [comment]
I don't see steps to creating tests on the repo. Maybe we add something clear in the Readme?

**Comment by m-mohr** [comment]
That's something relatively new, we should indeed update the Readme.

**Comment by DanielNwaeze** [comment]
what do we include for the data files for eurocrops datasets ec_fr and ec_si? no gpkgs are contained in the data.
do we convert the .shp file to .gpkg file? and wouldn't that consume a large amount of data?

**Comment by m-mohr** [comment]
We can create a subset of the data using ogr. See the test file for the command.

**Pull request merged**